app/mailers/user_mailer.rb

```diff
@@ -38,7 +38,7 @@ def invited_to_collection_notification(user_id, work_id, collection_id)
     @collection = Collection.find(collection_id)
     mail(
          to: @user.email,
-         subject: "[#{ArchiveConfig.APP_SHORT_NAME}]#{'[' + @collection.title + ']'} Request to include work in a collection"
+         subject: t("user_mailer.invited_to_collection_notification.subject", ao3: ArchiveConfig.APP_SHORT_NAME, collection: @collection.title)
```

Comment:
The Rails I18n guide says that [you can pass arguments to the translation of the subject](https://guides.rubyonrails.org/i18n.html#translations-for-action-mailer-e-mail-subjects) by calling e.g. `default_i18n_subject(user: user.name)`. Would that work here?

Reply:
Possibly, but that would be something to change throughout user_mailer, probably, for consistency - do we want to do that? I guess it saves a little bit of typing, but I can't see a major advantage in it.

Reply:
I like things that save typing. Reduces the chance of typos, or copy/paste errors.

(I figured you might be up for changing it throughout the mailer because you're already fixing the single quotes in a bunch of the subject lines, but it can go in a different PR.)

Reply:
Yeah, on second thoughts it does save significant space and risk of error, I'll do it.